go-controller/pkg/ovn/address_set.go

```diff
@@ -0,0 +1,259 @@
+package ovn
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"sync"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+
+	kapi "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+func (oc *Controller) getAddressSet(name string) *addressSet {
```

Comment:
Minor organizational nit: These functions would be better in `ovn.go` because they are specific to the cache attached to the `Contoller` struct.